extensions/Worker.Extensions.Kafka/release_notes.md

```diff
@@ -4,6 +4,6 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.Kafka <version>
+### Microsoft.Azure.Functions.Worker.Extensions.Kafka 3.9.0
```

Comment:
Updated. @liliankasem Is the dotnet worker team automatically releasing the extension after a release_notes upgrade? Can we modify the release pipeline to include permission from the extension owners?

Reply:
It's not automatic, a member of the team does the releases manually when there is a new WebJobs version. We can potentially discuss moving the worker extension so that it lives in the same repo as the WebJobs extension, this is something we're working to do with the ServiceBus/EventHubs teams (timeline tbd)